util/src/jvmTest/kotlin/care/data4life/sdk/util/Base64Test.kt

```diff
@@ -184,6 +184,8 @@ class Base64Test {
         private const val DECODED = "String to encode"
         private const val ENCODED = "U3RyaW5nIHRvIGVuY29kZQ=="
 
+        private const val DECODED_NON_ASCII = ""
```

Comment:
The non ASCII is not tested for Jvm and Android

Reply:
You are absolutely right - I overlooked at due to my starter problems with native. Sry. However ideally we have only one testset wich lives in common, but due to Roboelectric this will not work.

Reply:
done

Reply:
Base64 for Android is implemented on system level, so it has a hard dependency on beeing executed on Android. Which is sadly preventing common tests